app/games-list/page.tsx

```diff
@@ -0,0 +1,21 @@
+import Filter from "@/components/FilterGamesList";
+import GameCard from "@/components/GameCard";
+
+export default function GamesList() {
+  return (
+    <div className="flex flex-col sm:flex-row sm:items-start gap-2 items-center py-4">
+      <div className="w-[375px]">
+        <Filter />
```

Comment:
For the bgg list you dont need Filter component, you can discard/remove it for now